src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/EETypeVTableNode.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.TypeSystem;
+using System;
+using System.Collections.Generic;
+using Debug = System.Diagnostics.Debug;
+
+namespace ILCompiler.DependencyAnalysis
+{
+    internal class EETypeVTableNode : EmbeddedObjectNode
```

Comment:
Nit: We're calling this VTableNode and put it in a VTable collection in NodeFactory, but it's more like a VTableSlice.